core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_responses.html

```diff
@@ -30,7 +30,8 @@
                                on-delete-fn="deleteAnswerGroup"
                                outcome="answerGroup.outcome"
                                labelled-as-correct="answerGroup.labelledAsCorrect"
-                               num-rules="answerGroup.rules.length">
+                               num-rules="answerGroup.rules.length"
+                               is-correctness-feedback-enabled="isCorrectnessFeedbackEnabled()">
```

Comment:
See above -- no need to pass in new attrs. Maybe this means you don't even need to change StateResponses.js.

Reply:
Yep. Now , there is no such argument because the directives use the explorationCorrectnessFeedbackService directly.
Done!